src/components/AlertMessage/style.css

```diff
@@ -0,0 +1,273 @@
+@import url("https://fonts.googleapis.com/css?family=Lato:400,700");
+html {
+  display: grid;
+  min-height: 100%;
+}
+
+body {
```

Comment:
BEM Notation required. I do think we can make this a separate issue for someone else to work on?